guides/common/modules/proc_acd_installation.adoc

```diff
@@ -45,6 +45,31 @@ endif::[]
 # {foreman-maintain} service restart
 ----
 
+[id="{context}_installation_{SmartProxyServer}"]
+= Installing ACD on {SmartProxyServer}
+
+You need to install two packages to use ACD on a {SmartProxy}.
```

Comment:
```suggestion
You need to install two packages to use ACD on {SmartProxy}.
```

Reply:
You could always use "your" to get around it if you think it sounds weird!

Reply:
Fixed.